yt/visualization/volume_rendering/scene.py

```diff
@@ -511,11 +513,9 @@ def _show_mpl(self, im, sigma_clip=None, dpi=100):
         ax.set_position([0, 0, 1, 1])
 
         if sigma_clip is not None:
-            nz = im[im > 0.0]
-            nim = im / (nz.mean() + sigma_clip * np.std(nz))
```

Comment:
just a note to reviewers: this was the source of the bug. It should only normalize based on the RGB channels (i.e., `im[:,:,:3]`)

Reply:
awesome.  Nice catch.  I never got around to digging deeper, so I'll happily review your PR ;)